### PR TITLE
Update Input_Output.mdx - stdin and stdout Python

### PR DESCRIPTION
**Pull request by Garlicbread12**
Added stdin and stdout as methods for input and output for the Python section. I might additionally add stdin and stdout to Fast I/O for Python in the future if it would be helpful. Any tips relating to GitHub, as you could probably tell, would be very much appreciated :).

### CODE REVIEW

**Review by dongliu0426 — APPROVED** [review]
:clap: :clap:

File: content/1_General/Input_Output.mdx [hunk 1]
```diff
@@ -384,26 +384,52 @@ faster method of input.
...
 <PySection>
 
-### Python
-
+### Method 1 - [`input()`](https://docs.python.org/3/library/functions.html#input) and [`print()`](https://docs.python.org/3/library/functions.html#print)
 The most intuitive way to do input/output is using the built in
-[`input()`](https://docs.python.org/3/library/functions.html#input) and
-[`print()`](https://docs.python.org/3/library/functions.html#print) methods. The
+`input()` and
+`print()` methods. The
```
Comment (SansPapyrus683) [inline content/1_General/Input_Output.mdx:390]: wait why'd you remove the doc links

File: content/1_General/Input_Output.mdx [hunk 2]
```diff
@@ -412,10 +438,14 @@ along with [`map`](https://docs.python.org/3/library/functions.html#map) or a
 to read in multiple integers on the same line (separated by whitespace).
 
 ```py
+import sys
```
Comment (SansPapyrus683) [inline content/1_General/Input_Output.mdx:441]:
```suggestion
import sys

```
do this for the other imports as well

File: content/1_General/Input_Output.mdx [hunk 3]
```diff
@@ -426,20 +456,21 @@ integers.
...
-```
-
-<IncompleteSection>
-
-is there some other source which covers this?
 
-</IncompleteSection>
+# stdin and stdout
+a,b,c = map(int, sys.stdin.readline().split())
+print("sum is",a+b+c)
```
Comment (SansPapyrus683) [inline content/1_General/Input_Output.mdx:472]: spaces between ops

File: content/1_General/Input_Output.mdx [hunk 4]
```diff
@@ -384,26 +384,52 @@ faster method of input.
...
+my_int = int(input())
+# sys.stdout.write requires you to format code manually
+print(str(my_int) + '\n')
+
+# for larger inputs, you can buffer the problem's input, and sort through it yourself. Fastest input method for python.
+all_data = sys.stdin.read().split('\n')
+
+
+
+
```
Comment (SansPapyrus683) [inline content/1_General/Input_Output.mdx:432]:
```suggestion
```

**Review by SansPapyrus683 — CHANGES_REQUESTED** [review]

File: content/1_General/Input_Output.mdx (hunk 1 above)
Comment (Garlicbread12) [reply content/1_General/Input_Output.mdx:390]: I'm pretty sure I added them above.

File: content/1_General/Input_Output.mdx [hunk 5]
```diff
@@ -384,63 +384,92 @@ faster method of input.
...
 # prints integer n with " test" after it
 print(n, end=" test")
 ```
+### Method 2 - [`stdin`](https://docs.python.org/3/library/sys.html#sys.stdin) and [`stdout`](https://docs.python.org/3/library/sys.html#sys.stdout)
+
+The first method of reading input can be far slower than using `stdin` (Hundreds of times slower!). Coupled with python's relatively slow execution speed, reading input efficiently becomes incredibly important. 
+
+```py
+# import the sys module to use std in/out
+import sys
```
Comment (SansPapyrus683) [inline content/1_General/Input_Output.mdx:414]: put a newline after the import

File: content/1_General/Input_Output.mdx [hunk 6]
```diff
@@ -384,63 +384,92 @@ faster method of input.
...
+So taking three integers as input and printing their sum is quite simple. On a larger scale (thousands of integers), using stdin and stdout becomes far more important for speed:
 
 ```py
+import sys
 a,b,c = map(int, input().split())
+print("sum is",a + b + c)
+
+# stdin and stdout
+a,b,c = map(int, sys.stdin.readline().split())
 print("sum is",a+b+c)
```
Comment (SansPapyrus683) [inline content/1_General/Input_Output.mdx:470]: spaces between ops all over here as well
```suggestion
a, b, c = map(int, input().split())
print("sum is", a + b + c)

# stdin and stdout
a, b, c = map(int, sys.stdin.readline().split())
print("sum is", a + b + c)
```

**Review by SansPapyrus683 — CHANGES_REQUESTED** [review]

**Review by SansPapyrus683 — APPROVED** [review]
poggers

**Review by ryanchou-dev — APPROVED** [review]

**Pull request merged**